mw-webapp/src/component/text/Text.cy.tsx

```diff
@@ -0,0 +1,26 @@
+import {Text} from "src/component/text/Text";
+
+const TEXT_CY = "text";
+const TEXT_VALUE = "Hello World";
+
+describe("Text component", () => {
+  it("should render text when it is provided", () => {
+    cy.mount(
+      <Text
+        text={TEXT_VALUE}
+        cy={TEXT_CY}
+      />,
+    );
+    cy.get(`[data-cy=${TEXT_CY}]`).should("contain.text", TEXT_VALUE);
+  });
```

Comment:
```suggestion
    cy.get(getDataCy(TEXT_CY)).should("contain.text", TEXT_VALUE);
  });
```

Reply:
I noticed in cypress a test for the proposed